src/bridge.js

```diff
@@ -1,8 +1,22 @@
 import { isValidVersion } from './utils'
 
 export default class Bridge {
-  constructor () {
-    this.setAlpine(window.Alpine) // eslint-disable-line no-undef
+  init () {
+    const initAlpine = window.deferLoadingAlpine || ((callback) => callback())
+    window.deferLoadingAlpine = (callback) => {
+      this.setAlpine(window.Alpine) // eslint-disable-line no-undef
+
+      document.addEventListener('turbolinks:load', () => {
+        // Tag all cloaked elements on first page load.
+        this.tagCloakedElements(document.body)
```

Comment:
Tagging the cloaked elements needs to happen after the dom is ready, I just used `turbolinks:load` but we could use an async await for dom ready too (like Alpine has in utils).

Alpine itself needs to be initialized after we've captured the cloaked elements, in the test (which was correctly failing before and passes now) Alpine will initialize components (which strips the `x-cloak` attribute as soon as it starts, which can happen before any other events are fired.

I feel like there is still a better way to do this where we can listen for some DOM ready event to do the tagging, and still initialize Alpine outside of that handler but I'm not seeing it yet.